src/java.base/macosx/classes/sun/nio/ch/KQueue.java

```diff
@@ -47,74 +58,103 @@ private KQueue() { }
      *        void            *udata;         // opaque user data identifier
      * };
      */
-    static {
-        IOUtil.load();
-    }
-    private static final int SIZEOF_KQUEUEEVENT    = keventSize();
-    private static final int OFFSET_IDENT          = identOffset();
-    private static final int OFFSET_FILTER         = filterOffset();
-    private static final int OFFSET_FLAGS          = flagsOffset();
 
     // filters
-    static final int EVFILT_READ  = -1;
-    static final int EVFILT_WRITE = -2;
+    static final int EVFILT_READ  = kqueue_h.EVFILT_READ();
```

Comment:
Could we remove these constants and use `kqueue_h` constants directly?

Reply:
No, because those are `private`, accessible only through the `public static` getters.